### DISCUSSION TITLE
[讨论] flomo、memos的这种列表式的标签管理方式到底好不好？

### DISCUSSION THREAD

**Discussion by rebron1900** [usememos/memos#23] [General]
如题，目前我有不到八个标签，标签列表已经很长了，如果以后有几十上百个如何管理？
所以想和大家讨论一下这种列表形式到底好不好？

如果换成标签云的形式会不会好一些？或者有没有其他更具美感和更方便的标签管理形式呢。

![image](https://user-images.githubusercontent.com/7929884/149872600-bca78191-8d3b-4bfb-b280-a696d6feaaa2.png)

**Comment by rebron1900** [comment]
常用标签、快速检索这种，只是类似收藏、置顶的功能，还是希望能有更优雅的方式。

**Comment by bfansheng** [comment]
我很支持标签云的方式，我个人非常喜欢标签云，因为这可以让我很直观得看到所有标签，以及每个标签的使用频率，也会让自己很有成就感。
